src/lib/components/CountdownWidget.svelte

```diff
@@ -18,33 +18,31 @@
 	});
 </script>
 
-<div class="flex flex-row gap-4 {className}">
```

Comment:
The surrounding div has been removed, is it intended?

Reply:
reverted